images/access_setup/content/manifests/role.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
as discussed it may be fine for now but rights will need to be tighten

Reply:
Agreed.